tests/utils/fakeredis_ext.py

```diff
@@ -40,9 +42,9 @@ def _parse_pipeline_response_from_connection(self, connection):
         - QUEUED : acknowledges a command has been queued. There will be one per command sent.
         - LIST : list of responses
         """
-        # pop off the first command, which should be MULTI to signal start of transaction
         cmd = self.connection._sock.data.pop(0)
-        assert cmd[0] == 'MULTI'
+        if cmd[0] != 'MULTI':
```

Comment:
These conversions from assert to conditional exceptions are meant to appease bandit.